lib/fruitstrap.js

```diff
@@ -0,0 +1,52 @@
+import { exec } from 'teen_process';
+import logger from './logger';
+
+const FRUITSAP_PATH = 'fruitstrap';
+
+class Fruitstrap {
+ 
+  constructor (udid, fruitstrapPath) {
+    this.udid = udid;
+    this.fruitstrapPath = fruitstrapPath || FRUITSAP_PATH;
```

Comment:
I don't think there is a way.

You could _try_ returning a promise. But then will the Fruitstrap object ever be created? I don't think it would.

It would be better to add a method like `verify` which can be called right after the constructor. Or, if you want it all handled internally, you could keep track of the verification status, and call a method each time one of the other methods is called...